config/sync/field.field.config_pages.stanford_local_footer.su_local_foot_social.yml

```diff
@@ -16,12 +16,12 @@ field_name: su_local_foot_social
 entity_type: config_pages
 bundle: stanford_local_footer
 label: 'Social Links'
-description: ''
+description: 'Links added in this section will automatically create icons that will appear in your footer. Supported social links include: Bluesky, Facebook, Flickr, Github, Google Scholar, Instagram, LinkedIn, Mastodon.social, Threads, X, and YouTube.'
 required: false
 translatable: false
 default_value: {  }
 default_value_callback: ''
 settings:
   title: 2
-  link_type: 17
+  link_type: 16
```

Comment:
Why is the link type changing?

Reply:
Normally I would agree for a new field. But if someone put a local url, this could cause issues if a site has a local url. We can't make the change without first assuring it won't break the form or the display.

Reply:
Ok. Would trying it out locally be a sufficient test? What do you think would be the best way to test this?

Reply:
Yeah that would be fine. put in a variety of urls with the "Internal AND external" setting. Save, clear caches, verify the display works. Change to "External only", clear caches, verify the display and the form work. Try to save the form without any changes.

Reply:
@pookmish What do you think of what I described above? Should the link type stay as it was instead of external only?

Reply:
I did a query on our sites, and there are many sites that are not using external links. I'm not sure why. But since they aren't showing on the FE we should also add some validation to check for only the styles we have accounted for. Let me add that into this PR.